src/api/Controllers/BoreholeGeometryController.cs

```diff
@@ -32,13 +32,26 @@ public BoreholeGeometryController(BdmsContext context, ILogger<BoreholeGeometryE
         this.boreholeLockService = boreholeLockService;
     }
 
+    /// <summary>
+    /// Get borehole geometry data for the provided borehole id.
+    /// </summary>
+    /// <param name="boreholeId">The id of the <see cref="Borehole"/> to get geometry data for.</param>"
+    /// <returns>An IEnumerable of type <see cref="BoreholeGeometryElement"/>.</returns>
     [HttpGet]
     [Authorize(Policy = PolicyNames.Viewer)]
-    public async Task<IEnumerable<BoreholeGeometryElement>> GetAsync([FromQuery] int boreholeId)
+    public async Task<IActionResult> GetAsync([FromQuery] int boreholeId)
     {
-        return await GetBoreholeGeometry(boreholeId).ConfigureAwait(false);
+        if (await boreholeLockService.IsUserLackingPermissions(boreholeId, HttpContext.GetUserSubjectId()).ConfigureAwait(false))
+            return Unauthorized("You are missing permissions for the borehole with the requested id.");
```

Comment:
REQ: Einfach `Unauthorized()` ohne Text zurückgeben, das ist einfacher und man muss nicht überall den Text in Sync halten.